results/customer-portal/installer-platform.detailed-description.html

```diff
@@ -0,0 +1,3 @@
+The Red Hat JBoss Developer Studio Platform Installer includes everything you need to start working with the Red Hat Container Development Kit and Red Hat JBoss Developer Studio, including OpenJDK 8, Cygwin, PuTTY, and OpenShift Client (OC) Tools. 
+
+It requires Vagrant 1.7.4 and Virtual Box 5.0.8 or later, but will install them for you if not already present on your 64-bit Windows system if a network is available.
```

Comment:
Vagrant 1.7.4 is recommended because 1.8.0 and 1.8.1 have some network issues. But it doesn't mean that an older vagrant won't work. I believe it just wasn't tested on an older versions.
And CDK was tested on VirtualBox 5.0.x. and works fine. And again we don't know if it works on older versions.  
I'm OK if you keep these versions. I just not sure it's worth being so detailed here.